internal/service/cloudprovideraccess/resource_cloud_provider_access_authorization_test.go

```diff
@@ -75,7 +75,7 @@ resource "aws_iam_role_policy" "test_policy" {
     "Version": "2012-10-17",
     "Statement": [
       {
-        "Effect": "Allow",
+        "Effect": "Deny",
```

Comment:
[q] is it that these policies are never actually used for action executions that's why we are changing this to "Deny"?

Reply:
For this one yes, for the context of the test itself there is no need to have permissions.